### DISCUSSION TITLE
GPG error when apt update

### DISCUSSION THREAD

**Discussion by ericknyoto** [ddev/ddev#5300] [Q&A] [answered]
I got this error when sudo apt update. How do I fix that?
```
W: GPG error: https://pkg.ddev.com/apt * InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY C16711FC1179C7CA
E: The repository 'https://pkg.ddev.com/apt * InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**Comment by rfay** [comment, marked as answer]
Hi @ericknyoto - First of all, did you install the key properly? See https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#linux - I recommend that you just do it again. Install the key, install the /etc/apt/sources.list.d/ddev.list

If you're on WSL2, use the [WSL2 install script](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#windows) again.

Second, it may be that you're behind a VPN which alters certificates as they go through. That's more complicated, but it would break several other things.

**Reply by ericknyoto** [reply]
Thank you! I will try it.
